app/assets/stylesheets/_forms.scss

```diff
@@ -0,0 +1,18 @@
+input,
```

Comment:
Be aware that this will apply to `input[type="checkbox"]` & `input[type="radio"]`, as well